app/src/main/java/com/babylon/wallet/android/presentation/transfer/TransferViewModel.kt

```diff
@@ -367,6 +412,13 @@ class TransferViewModel @Inject constructor(
             data object None : Message
             data class Added(val message: String = "") : Message
         }
+
+        data class MaxAmountMessage(
+            val emptyAccountAmount: BigDecimal,
```

Comment:
I don't understand the `empty*` meaning in this class.

Reply:
Also this class does not need the `emptyAccountAmount`. You can still have the maxAccountAmount and you can define a property inside like `val amountWithoutFees: BigDecimal` which will return the `maxAccountAmount - BigDecimal.ONE`

Reply:
Ah yes fair enough we can do 👍